tests/risks/test_data_transformation.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+import numpy as np
+
+import pytest
+
+from vivarium.testing_utilities import build_table
+from vivarium.framework.configuration import build_simulation_configuration
+from vivarium_public_health.risks.data_transformation import *
+
+def test_should_rebin():
+    test_config = build_simulation_configuration()
+    test_config['population'] = {'population_size': 100}
+    assert not should_rebin('test_risk', test_config)
+
+    test_config['test_risk'] = {}
+    assert not should_rebin('test_risk', test_config)
+
+    test_config['test_risk'].rebin = False
+    assert not should_rebin('test_risk', test_config)
+
+    test_config['test_risk']['rebin'] = True
+    assert should_rebin('test_risk', test_config)
+
+
```

Comment:
We generally need to take better advantage of pytest fixtures and parameterization.  A lot of tests look like this where we're mixing the generation of test data with the logic of the test.  That makes them very hard to maintain in the long run.